figshare/figshare.py

```diff
@@ -290,3 +293,37 @@ def get_file_details(self, article_id, file_id):
         response = issue_request('GET', url,
                                  headers=self.get_headers(token=self.token))
         return response
+
+    def retrieve_files_from_article(self, article_id, outpath=''):
```

Comment:
use `None` as default for outpath. Let's call this arg `directory` instead.

Reply:
Done and done.